### PR TITLE
es6-module exercise - Matthew Kweon

### PR DESCRIPTION
**Pull request by MattKweon**
Kapture 2022-10-21 at 11 31 33

### CODE REVIEW

File: es6-modules/src/lib/create-element.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  toArray(children).forEach(child => {
+    if (!(child instanceof HTMLElement)) {
+      child = document.createTextNode(child);
+    }
+    parent.appendChild(child);
+  });
+  return parent;
+}
+
+module.exports = createElement;
```
Comment (thebearingedge) [inline es6-modules/src/lib/create-element.js]: Good work so far, check the modules in the `lib` directory too

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by MattKweon** [comment]
Screen Shot 2022-10-21 at 11 38 02 AM
Screen Shot 2022-10-21 at 11 38 16 AM
Screen Shot 2022-10-21 at 11 38 24 AM

**Review by thebearingedge — CHANGES_REQUESTED** [review]
I don't think this actually builds properly

**Comment by MattKweon** [comment]
Kapture 2022-10-21 at 12 12 12
Screen Shot 2022-10-21 at 12 12 30 PM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**